package.xml

```diff
@@ -1,31 +1,34 @@
-<package>
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>srdfdom</name>
   <version>0.5.1</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
   <author email="isucan@willowgarage.com">Ioan Sucan</author>
   <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
+  <author email="vmayoral@acutronicrobotics.com">Víctor Mayoral Vilches</author>
```

Comment:
i don't think its fair to claim ownership over a package just because you ported it to ROS 2. for example, i have 5 commits and @rhaschke 17 but we haven't made ourselves authors of the package.

feel free to list yourself as a maintainer though!